api/models/Users.models.ts

```diff
@@ -1,7 +1,27 @@
-const mongoose = require("mongoose");
-const bcrypt = require("bcrypt");
+import mongoose, { Document, Schema } from "mongoose";
+import bcrypt from "bcrypt";
 
-const UsersSchema = mongoose.Schema({
+export interface UserEntity extends Document {
+  fullname: string;
```

Comment:
Validate with frontend. This is different. Different between frontend and backend is muy malo.